File: Homeworks/homework2/torun_boran.dart
void main() { 
  for(var i = 0; i < 5; i++)
  {
    print(i);
  }
  
  print("--------------------");
    
  // break 
  for(var i = 0; i < 5; i++)
  {
    print(i);
    break;
  }
  
   print("--------------------");
  
  goto1:
  for(var i = 0; i < 5; i++)
  {
    print(i);
    break goto1;
  } 
  
  print("--------------------");
  
  goto2:
  for(var i = 0; i < 5; i++)
  {
    for(var j = 0; j < 5; j++)
    {
      print("i = " + i.toString() + " j = " + j.toString());
      if(i == 2)
      {
        break goto2;
      }
    }
  }
  
  print("--------------------");
  
  // continue 
  for(var i = 0; i < 5; i++)
  {
    print(i);
    continue;
  }
   
  print("--------------------");
  
  goto3:
  for(var i = 0; i < 5; i++)
  {
    if(i == 2)
    {
      continue goto3;
    }
    print(i);
  }  
  
  print("--------------------");
  goto4:
  for(var i = 0; i < 5; i++)
  {
    for(var j = 0; j < 5; j++)
    {
      if(i == 2)
      {
        continue goto4;
      }
      print("i = " + i.toString() + " j = " + j.toString());
    }
  }
}

File: Homeworks/homework2/torun_boran.rs
fn main() {
    for i in 1..10{
        print!("i = {}, ", i);
    }
    
    println!("\n");
    
    // break
    for i in 1..10{
        if i == 6
        {
            break;
        }
        print!("i = {}, ", i);
    }
    
    println!("\n");
    
    for i in 1..5{
        for j in 1..5{
            if i == 3
            {
                break;
            }
            println!("i = {}, j = {}", i, j);
        }
    }
    
    println!("\n");
    
    'goto1: loop
    {
        for i in 1..5{
            if i == 3{
                break 'goto1;
            }
            print!("i = {}, ", i);
        }
    }
    
    println!("\n");
    
    'goto2: loop
    {
        for i in 1..5{
            for j in 1..5{
                if i == 3{
                    break 'goto2;
                }
                println!("i = {}, j = {}", i, j);
            }
        }
    }
    
    println!("\n");
    
    // continue
    for i in 1..5{
        if i == 3{
            continue;
        }
        print!("i = {}, ", i);
    }
    
    println!("\n");
    
    'goto3:
    for i in 1..5{
        if i == 3{
            continue 'goto3;
        }
        print!("i = {}, ", i);
    }
    
    println!("\n");
    
    'goto4:
    for i in 1..5{
        for j in 1..5{
            if i == 3{
                continue 'goto4;
            }
            println!("i = {}, j = {}", i, j);
        }
    }
    
}